akka-cluster/src/multi-jvm/scala/akka/cluster/artery/ArterySunnyWeatherSpec.scala

```diff
@@ -1,18 +1,19 @@
 /**
  * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster
+package akka.cluster.artery
```

Comment:
Äsch. My intention was to copy. Will try to make it a config instead now that I know that I didn't have to change anything but the config.